### DISCUSSION TITLE
Internal Testing Track Name for Instant Apps

### DISCUSSION THREAD

**Discussion by predatorx7** [codemagic-ci-cd/codemagic-docs#2726] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

1. Guessing names for instant app's internal testing track name, `instant_app:qa`. Could not find the right track name to use for instant apps.
2. Attempting to publish to internal testing track for instant apps. 

### Expected results

App to be published on internal testing app for instant app (releaseType=instant)

### Actual results

Getting error

```json
Publishing failed :|
Google Play failed to upload artefacts. Track not found: instantapp:qa.: {
    "error": {
        "code": 404,
        "message": "Track not found: instantapp:qa.",
        "status": "NOT_FOUND"
    }
}
```

### Build id (optional)

66df15d8eb428006efc4aa8f

### More

1. How to add the same app bundle uploaded to a track to instant app. 
2. Codemagic right now doesn't have an option to upload or add from library an app bundle for instant app

**Comment by dtrdic** [comment, marked as answer]
hey @predatorx7 yes unfortunately you can't automatically publish the binary to this instant-apps tracks, you will have to do it manually.
But I'll classify this thread as a feature request so the team will take a look at it as a future improvement
